lnbits/core/tasks.py

```diff
@@ -73,16 +67,6 @@ async def register_watchdog():
     # watchdog = asyncio.create_task(watchdog_task())
 
 
-async def unregister_watchdog():
```

Comment:
- since `watchdog` was also converted please adapt the commented code to the new paradigm

https://github.com/lnbits/lnbits/blob/2928209659950f27cadd4701f5a10c84a75efe2b/lnbits/core/tasks.py#L67